server/re/src/icatAdminMS.cpp

```diff
@@ -633,82 +633,6 @@ int msiRenameLocalZoneCollection(msParam_t* _new_zone_name, ruleExecInfo_t* _rei
     return SYS_SERVICE_ROLE_NOT_SUPPORTED;
 }
 
-/**
- * \fn msiAclPolicy(msParam_t *msParam, ruleExecInfo_t *)
- *
- * \brief   When called (e.g. from acAclPolicy) and with "STRICT" as the
- *    argument, this will set the ACL policy (for GeneralQuery) to be
- *    extended (most strict).
- *
- * \module core
- *
- * \since pre-2.1
- *
- *
- * \note Should not be used outside of the rules defined in core.re.
- * Once set STRICT, strict mode remains in force (users can't call it in
- * another rule to change the mode back to non-strict).
- * See core.re.
- *
- * \usage See clients/icommands/test/rules/
- *
- * \param[in] msParam - a msParam of type STR_MS_T - can have value 'STRICT'
- * \param[in,out] - The RuleExecInfo structure that is automatically
- *    handled by the rule engine. The user does not include rei as a
- *    parameter in the rule invocation.
- *
- * \DolVarDependence
- * \DolVarModified
- * \iCatAttrDependence
- * \iCatAttrModified
- * \sideeffect none
- *
- * \return integer
- * \retval status
- * \pre none
- * \post none
- * \sa none
- **/
-int
-msiAclPolicy( msParam_t* msParam, ruleExecInfo_t* ) {
-    char *inputArg;
-
-    std::string svc_role;
-    irods::error ret = get_catalog_service_role(svc_role);
-    if(!ret.ok()) {
-        irods::log(PASS(ret));
-        return ret.code();
-    }
-
-    std::string strict = "off";
-    inputArg = ( char * ) msParam->inOutStruct;
-    if ( inputArg != NULL ) {
-        if ( strncmp( inputArg, "STRICT", 6 ) == 0 ) {
-            if( irods::KW_CFG_SERVICE_ROLE_PROVIDER == svc_role ) {
-                chlGenQueryAccessControlSetup( NULL, NULL, NULL, 0, 2 );
-                strict = "on";
-            }
-        }
-    }
-    else {
-        if( irods::KW_CFG_SERVICE_ROLE_PROVIDER == svc_role ) {
-            chlGenQueryAccessControlSetup( NULL, NULL, NULL, 0, 0 );
-        }
-    }
-
-    // set a strict acl prop
-    try {
-        irods::set_server_property<std::string>(irods::STRICT_ACL_KW, strict);
```

Comment:
Not sure if we're allowed to remove it, but I see `DISABLE_STRICT_ACL_KW`. Should stop supporting this in the server and/or remove it, or at least check to see how it's being used?

Reply:
yes, please let's not support something with that name

Reply:
Removed.